src/hydration.js

```diff
@@ -0,0 +1,36 @@
+class Hydration {
+  constructor(hydrationData, id) {
+    this.data = hydrationData;
+    this.id = id;
+    this.user = this.findUserHydroData();
+  };
+
+  findUserHydroData() {
+    return this.data.filter(user => user.userID === this.id)
+
+  }
+
+  calculateAvgOzsPerDay() {
```

Comment:
this method calculates the total Ozs for a data set and divides that number by the number of days in the data set to return the average ozs per day.

Reply:
Sure does. I think this works pretty well.